app/src/main/java/com/bachelor_group54/funnregistrering/ListAdapter.java

```diff
@@ -0,0 +1,109 @@
+package com.bachelor_group54.funnregistrering;
+
+import android.app.Activity;
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+
```

Comment:
Veldig bra kommentert her også! Eneste er at vi kanskje kan vurdere en kommentar for hele klassen.